plugins/jdg-nightly/pom.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.radargun</groupId>
+      <artifactId>radargun-plugin-parent</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+   </parent>
+
+   <artifactId>plugin-jdg-nightly</artifactId>
+   <name>JDG nightly build plugin for RadarGun</name>
+
+   <properties>
+      <version.infinispan>${jdg-nightly.infinispan.version}</version.infinispan>
+   </properties>
+
+   <profiles>
+      <profile>
+         <id>jdg-nightly-infinispan-version-default</id>
+         <activation>
+            <property>
+               <name>!jdg-nightly.infinispan.version</name>
+            </property>
+         </activation>
+         <properties>
+            <version.infinispan>8.5.0.ER3-redhat-1</version.infinispan>
+         </properties>
+      </profile>
+   </profiles>
+
+   <dependencies>
+       <dependency>
+         <groupId>org.radargun</groupId>
+         <artifactId>plugin-jdg71</artifactId>
+         <version>${project.version}</version>
+       </dependency>
+
+       <dependency>
+           <groupId>org.radargun</groupId>
+           <artifactId>plugin-infinispan91</artifactId>
```

Comment:
Do we need to depend on infinispan91 when we already depend on jdg71/jdg72 ?

Reply:
Nope, there should only be one here